app/views/user/requests/_request.html.erb

```diff
@@ -1,20 +1,18 @@
-<div class="card">
-  <div class="card-header">
-    <h3>Request for <%= t("documents.types.#{request.document_type}") %></h3>
-  </div>
+<div class="card card--with-fancy-footer birth-record card--">
```

Comment:
Where is this change coming from @JacOng17 - this isn't in Ross's designs or requested in the trello card